arches_lingo/src/arches_lingo/components/scheme/report/SchemeNote.vue

```diff
@@ -1,20 +1,154 @@
+const route = useRoute();
+const toast = useToast();
+const metaStringLabel: MetaStringText = {
+    deleteConfirm: $gettext("Are you sure you want to delete this note?"),
+    language: $gettext("Note Language"),
+    name: $gettext("Note Name"),
+    type: $gettext("Note Type"),
 };
 
-defineExpose({ save, getSectionValue });
-const { $gettext } = useGettext();
+withDefaults(
+    defineProps<{
+        mode?: DataComponentMode;
+        tileId?: string | null;
+    }>(),
+    {
+        mode: VIEW,
+        tileId: null, // editor arg specifying what tile to operate on.
+    },
+);
+
+const emits = defineEmits([OPEN_EDITOR]);
+
+defineExpose({ getSectionValue });
+
+onMounted(() => {
+    getSectionValue();
+});
+
+async function getSectionValue() {
+    try {
+        const result = await fetchSchemeNotes(route.params.id as string);
+        schemeInstance.value = {
+            statement: result.statement,
+        };
+    } catch (error) {
+        toast.add({
+            severity: ERROR,
+            summary: $gettext("Error"),
+            detail:
+                error instanceof Error
+                    ? error.message
+                    : $gettext("Could not fetch the notes for the resource"),
+        });
+    }
+}
+
+async function deleteSectionValue(tileId: string) {
+    let result = false;
+    try {
+        result = await deleteSchemeNoteTile(route.params.id as string, tileId);
+    } catch (error) {
+        toast.add({
+            severity: ERROR,
+            summary: $gettext("Error"),
+            detail:
+                error instanceof Error
+                    ? error.message
+                    : $gettext("Could not delete selected note"),
+        });
+    }
+
+    if (result) {
+        getSectionValue();
+    }
+}
+
+function editSectionValue(tileId: string) {
+    const schemeStatement = schemeInstance.value?.statement?.find(
+        (tile) => tile.tileid === tileId,
+    );
+    if (schemeStatement && schemeStatement.tileid === tileId) {
+        emits(OPEN_EDITOR, schemeStatement.tileid);
+    } else {
+        toast.add({
+            severity: ERROR,
+            summary: $gettext("Error"),
+            detail: $gettext("Could not find the selected label to edit"),
+        });
+    }
+}
 </script>
 
 <template>
-    <SchemeReportSection :title-text="$gettext('Scheme Notes')">
-        abc
-    </SchemeReportSection>
+    <div v-if="mode === VIEW">
+        <SchemeReportSection
+            :title-text="$gettext('Scheme Notes')"
+            @open-editor="emits(OPEN_EDITOR)"
+        >
+            <MetaStringViewer
+                :meta-strings="schemeInstance?.statement"
+                :meta-string-text="metaStringLabel"
+                @edit-string="editSectionValue"
+                @delete-string="deleteSectionValue"
+            >
+                <template #name="{ rowData }">
+                    {{ (rowData as SchemeStatement).statement_content_n1 }}
```

Comment:
I'm unsure, but I _think_ this needs `span` wrapping as well

Reply:
no objections.  🏓